src/metaswitch/crest/tools/bulk_create.py

```diff
@@ -177,9 +177,11 @@ def write_homer_scripts(csv_filename):
 
         for public_id, private_id, realm, password in csv_iterator(csv_filename):
             # Add the simservs document for the user to the documents table on Homer
+            # This SQL command is as safe from injection risk as the CSV file is.
+            # Exclude from Bandit security check as there's no workaround when
+            # we need to write the command to bash script.
```

Comment:
I don't completely understand this comment. Am I write in thinking that we're not actually executing this SQL command here, just writing it to a script? If so, let's just say "we're not actually executing this SQL command here, just writing it to a script, so exclude it from the Bandit security checks".

Reply:
Kk will do